iron-location.html

```diff
@@ -209,14 +209,16 @@
         // nothing to do, the URL didn't change
         return;
       }
+      // Need to use a full URL in case the containing page has a base URI.
+      var fullNewUrl = new URL(newUrl, window.location.href).href;
```

Comment:
If I do `someAnchor.href = 'foo'` I will get a path that is relative to the current `window.location.pathname`. So, I guess the question is which assignment semantics you expect. I agree that assigning to an `iron-location`'s properties should be more like assigning to `window.location`'s properties.